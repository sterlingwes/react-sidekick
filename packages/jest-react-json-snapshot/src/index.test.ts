import { simpleFixture } from "./fixtures/simple.fixture";

describe("jest react json snapshot serializer", () => {
  describe("simple tree", () => {
    it("should produce a JSON snapshot w/ callsite info", () => {
      expect(simpleFixture).toMatchSnapshot();
    });
  });
});
