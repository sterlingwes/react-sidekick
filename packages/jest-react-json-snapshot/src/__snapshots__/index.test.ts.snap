// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest react json snapshot serializer simple tree should produce a JSON snapshot w/ callsite info 1`] = `
Array [
  {
  "meta": {
    "source": "react-test-render-transformer",
    "callSite": {
      "filePath": "/packages/jest-react-json-snapshot/dist/index.js",
      "lineNumber": 22,
      "column": 11
    }
  },
  "tree": []
},
]
`;
